src/renderer/containers/Validator/Validator.tsx

```diff
@@ -89,13 +90,28 @@ export const Validator: React.FunctionComponent<IValidatorSimpleProps> = (props:
     };
 
     const renderValidatorButtons = (): React.ReactElement => {
-        if (!nodes.length) return null;
+        if (
+            !nodes.length ||
+            validator.status === ValidatorStatus.BEACON_NODE_OFFLINE ||
+            validator.status === ValidatorStatus.SYNCING_BEACON_NODE ||
+            validator.status === ValidatorStatus.WAITING_DEPOSIT ||
+            validator.status === ValidatorStatus.PROCESSING_DEPOSIT ||
+            validator.status === ValidatorStatus.PENDING_DEPOSIT_OR_ACTIVATION ||
+            validator.status === ValidatorStatus.ERROR
```

Comment:
you should probably create array of statuses which doesn't allow validator starting.
`array.includes(validator.status)`